### DISCUSSION TITLE
Lesson 9: make question

### DISCUSSION THREAD

**Discussion by RichardPalladino** [Cyfrin/foundry-full-course-cu#382] [Q&A] [answered]
Toward the end of Lesson 9, Patrick does "toggle word wrap" for the install stanza.  Why do this?  Why not just remove the semi-colon after the `install:` statement and make a multi-line stanza?

For example, instead of:
`install:; forge install Cyfrin/foundry-devops@0.0.11 --no-commit && forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit && forge install foundry-rs/forge-std@v1.5.3 --no-commit && forge install transmissions11/solmate@v6 --no-commit`

with word wrap, do:

```
install: 
    forge install Cyfrin/foundry-devops@0.0.11 --no-commit
    forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit
    forge install foundry-rs/forge-std@v1.5.3 --no-commit
    forge install transmissions11/solmate@v6 --no-commit
```

Is it just a style preference, or is there some kind of optimization with placing it all on a single, super-long line?

**Comment by ChefAharoni** [comment, marked as answer]
Just a styling preference, there's no actual difference how you write it.

**Comment by Niferu** [comment]
Hello @RichardPalladino 

@ChefAharoni is right if you write make code like you did in example here, but once you would like to format code by adding tabs/enters or smth and code would not be in same line it will simply break.
